common/pupil/screening.ts

```diff
@@ -27,3 +28,19 @@ export async function addPupilScreening(pupil: Pupil, screening: PupilScreeningI
 
     logger.info(`Added ${screening.status || 'pending'} screening for pupil ${pupil.id}`, screening);
 }
+
+interface PupilScreeningUpdate {
+    status: PupilScreeningStatus;
+    comment: string;
+}
+
+export async function updatePupilScreening(pupilScreeningId: number, screeningUpdate: PupilScreeningUpdate) {
+    const screening = await prisma.pupil_screening.findFirst({ where: { id: pupilScreeningId } });
+    if (screening === null) {
+        logger.error('cannot find pupil with', { id: pupilScreeningId });
+        throw new NotFoundError('pupil screening not found');
+    }
+
+    await prisma.pupil_screening.update({ where: { id: pupilScreeningId }, data: { ...screeningUpdate, updatedAt: new Date() } });
+    logger.debug('successfully updated pupil', { pupilScreeningId, screeningUpdate });
+}
```

Comment:
If the status changes to success or rejected we want to send an email to the user, so best already integrate `Notification.actionTaken` here.